lib/ansible/modules/cron.py

```diff
@@ -325,7 +325,7 @@ def write(self, backup_file=None):
             os.unlink(path)
 
             if rc != 0:
-                self.module.fail_json(msg=err)
+                self.module.fail_json(msg=err, stdout=out, rc=rc)
```

Comment:
Could you please add a changelog and test for this change?

Reply:
I would move err to stderr= and update msg= with ''failed to install new cronfile: {path}' or something similar and informative that does not rely on (mostly cryptic) OS errors.